screens/HomeTour/Carousel/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Fragment, useState, useRef } from 'react'
+import { ScrollView, View } from 'react-native'
+import { Dot } from './styled'
+import Slide from './slide'
+
+const Carousel = ({ slides }) => {
+    const slider = useRef(null)
+    const [slidesNumber, setSlideNumber] = useState(1)
+
+
+    return (
+        <Fragment>
+            <ScrollView
+                ref={slider}
+                horizontal
+                pagingEnabled
+                showsHorizontalScrollIndicator={false}
+                onScrollEndDrag={(e) => console.log(e.nativeEvent)}
+            >
+                {slides.map((item) => <Slide text={item.text} image={item.file} />)}
```

Comment:
List of elements require key

👉https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Bien vu!